### PR TITLE
[TR] PIM-4612: Fixed Quick Export on mongo

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | running
| Specs & Static     | Y
| Changelog updated  | Y

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Extension/MassAction/Util/ProductFieldsBuilder.php [hunk 1]
```diff
@@ -69,11 +69,16 @@ public function __construct(
...
     {
         $this->prepareAvailableAttributeIds($productIds);
-        $attributeRepo  = $this->productManager->getAttributeRepository();
-        $attributesList = $attributeRepo->findBy(array('id' => $this->getAttributeIds()));
-        $fieldsList     = $this->prepareFieldsList($attributesList);
 
-        return $fieldsList;
+        $attributes = $this->getAttributeIds();
+
+        if (empty($attributes)) {
```
Comment (willy-ahva) [inline src/Pim/Bundle/DataGridBundle/Extension/MassAction/Util/ProductFieldsBuilder.php]: I would prefer avoid a double return : 
if (empty($attr)) {
$attributes = [];
} else {
$attributes = $this->productManager->getAttributeRepository()->findBy(array('id' => $this->getAttributeIds()));
}

return $attributes;

Comment (ClementGautier) [inline src/Pim/Bundle/DataGridBundle/Extension/MassAction/Util/ProductFieldsBuilder.php]: I never use else statements in my code for readability:

* https://pear.php.net/manual/en/standards.bestpractices.php#standards.bestpractices.returnearly
* http://phpmd.org/rules/cleancode.html (see ElseExpression)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.3.22
```
Comment (grena) [inline CHANGELOG.md]: I think the script creates the version (you can leave 1.3.x here I guess)

**Comment by jjanvier** [comment]
GTM

**Comment by grena** [comment]
GTM

File: CHANGELOG.md (hunk 2 above)
Comment (ClementGautier) [inline CHANGELOG.md]: ok, I'll do that on the next versions.

**Pull request merged**